src/main/resources/assets/computercraft/lua/rom/programs/http/pastebin.lua

```diff
@@ -20,11 +20,20 @@ end
 
 local function get(paste)
     write( "Connecting to pastebin.com... " )
+    -- Add a cache buster so that spam protection is re-checked
+    local cacheBuster = tostring(math.random())
     local response = http.get(
-        "https://pastebin.com/raw/"..textutils.urlEncode( paste )
+        "https://pastebin.com/raw/"..textutils.urlEncode( paste ).."?cb="..cacheBuster
     )
 
     if response then
+        -- If spam protection is activated, we get redirected to /paste with Content-Type: text/html
+        local headers = response.getResponseHeaders()
+        if not headers["Content-Type"] or not headers["Content-Type"]:find( "^text/plain" ) then
+            printError( "\nFailed to get paste. Please complete the captcha in a web browser: https://pastebin.com/"..textutils.urlEncode( paste ) )
```

Comment:
Just a couple of things:
 - Can you use `io.stderr:write` instead? I'd quite like to move most programs over to use `io` in the future, so might as well make a start now.
 - Thoughts on making this print `Failed.`, and then the reason why on a new line? Could also do that for the `Failed.` below too.

Reply:
Yep